### ISSUE TITLE
π-cross: Site accessibility and mobile-friendly design checks

### ISSUE THREAD

**Issue by Brycey92** [AEFeinstein/Super-2023-Swadge-FW#309]
Since font sizes and things have been changed between https://picross.hdyar.com and https://swadge.com/picross/, we need to double-check that the site is still accessible to those with reduced vision, and that it works well on mobile.

**Comment by Brycey92** [comment]
We need alt text on the images describing what's actually there, so the site could still be used if images don't load or someone is having the site narrated

also we got this super cool thing https://wave.webaim.org/report#/https://swadge.com/picross

**Comment by Brycey92** [comment]
Accessibility isn't a nice-to-have, it should be a requirement. Updated labels accordingly

**Comment by AEFeinstein** [comment]
Making the instructions accessible for the visually impaired is incredibly noble, but the game itself is not accessible in that way, so while there is value, there may not be as much as you think.

**Comment by Brycey92** [comment]
Damn it 😂
